### ISSUE TITLE
Can't use in lists

### ISSUE THREAD

**Issue by SubZeroX** [Yeboster/autocomplete-obsidian#30]
When one try to use it in lists, the derised effect of autocomplete actually creates another bullet.

https://user-images.githubusercontent.com/13406988/141113004-d4e1e1fa-7594-48aa-a7e3-fc7343bde3fb.mp4

**Comment by garahad** [comment]
same for me too

**Comment by fabianski7** [comment]
up

**Comment by l2aggron** [comment]
Hi there! I had the same question, seems you have same problem, maybe this helps - https://github.com/Yeboster/autocomplete-obsidian/issues/22

**Comment by SubZeroX** [comment]
> Hi there! I had the same question, seems you have same problem, maybe this helps - #22

Yeah, that's the main problem, can't live without the outliner haha. Just found out about the "Various Complements" plugin that can also work with outliner and also have the ability to link to internal links
